src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -241,11 +228,21 @@
         featured-communities
         @view-type]])))
 
+(defn f-discover
+  []
+  (fn []
+    (rn/use-effect
```

Comment:
it's better to avoid using effects and f component, and just dispatch outside render function